packages/directlinespeech/src/createWebSpeechPonyfillFactory.js

```diff
@@ -21,6 +21,14 @@ export default function({
   recognizer,
   textNormalization
 }) {
+  if (!ponyfill.AudioContext) {
+    console.warn(
+      'botframework-directlinespeech-sdk: This browser does not support Web Audio API. Speech support is disabled.'
```

Comment:
Do you want to use the which browser util to specify with browser it is? Not a big deal if you want to keep it as is.

Reply:
Prefer keep it as-is.